app/js/controllers/main.js

```diff
@@ -70,6 +78,23 @@ export default class MainController extends BaseController {
   }
 
   /**
+   * Clear all data/settings stored on the browser. Use with caution.
+   *
+   * @param {boolean} ignoreServiceWorker
+   * @return {Promise}
+   */
+  clear(ignoreServiceWorker = true) {
+    const promises = [this[p.settings].clear()];
+
+    if (!navigator.serviceWorker && !ignoreServiceWorker) {
+      promises.push(navigator.serviceWorker.ready
+        .then((registration) => registration.unregister()));
+    }
+
+    return Promise.all(promises);
+  }
```

Comment:
I wanted to create the settings instance in main.js; so it makes sense to clear it from main too.

about service workers: it's not clear to me that "server" should hold things about service workers... it's not server-related at all ? but I don't really mind in the end.